packages/@lwc/integration-karma/test/api/getComponentDef/index.spec.js

```diff
@@ -152,6 +153,26 @@ describe('@api', () => {
             childMethod: PublicMethodsInheritance.prototype.childMethod,
         });
     });
+
+    it('should log warning when accessing a private prop', () => {
+        const message = `Error: [LWC warn]: The property "privateGetter" is not publicly accessible. Add the @api annotation to the property declaration or getter/setter in the component to make it accessible.`;
+        const elm = createElement('x-private-accessor', { is: PrivateAccessors });
+        document.body.appendChild(elm);
+
+        expect(() => {
+            elm['privateGetter'];
+        }).toLogWarningDev(message);
+    });
```

Comment:
This test looks great to me. It would also be great to have tests for:

- Setter
- A property that is not declared as a getter/setter (e.g. `class { foo = 'bar' }`)
- A `@track`ed property (as mentioned elsewhere, we probably still want to warn on this)